server/services/visitSessionsService.ts

```diff
@@ -216,6 +216,16 @@ export default class VisitSessionsService {
     return this.buildVisitInformation(visit)
   }
 
+  async getBookedVisit({ username, reference }: { username: string; reference: string }): Promise<VisitInformation> {
+    const token = await this.systemToken(username)
+    const visitSchedulerApiClient = this.visitSchedulerApiClientBuilder(token)
+
+    logger.info(`Get visit ${reference}`)
```

Comment:
Only change I can see is the log message 👍

Reply:
Annoyingly I did consider that, then wasn't sure if it made sense from a log perspective!